bptestrunner/bluepill_batch_test_runner.template.sh

```diff
@@ -29,8 +27,11 @@ mkdir $BP_WORKING_FOLDER
 for test_bundle in ${TEST_BUNDLE_PATHS[@]}; do
     if [[ $test_bundle == *.zip ]]; then
         tar -C $BP_WORKING_FOLDER -xzf $test_bundle
+    elif [[ $test_bundle == *.xctest ]]; then
+        cp -cr $test_bundle $BP_WORKING_FOLDER
+        chmod -R 777 "$BP_WORKING_FOLDER/$(basename "$test_bundle")"
```

Comment:
Can we change this to?

```suggestion
        chmod -R ug+w "$BP_WORKING_FOLDER/$(basename "$test_bundle")"
```

It seems that setting 777 for all files and directories will leave non-executable files as executable and open up permissions too much